eng/pipelines/installer/jobs/base-job.yml

```diff
@@ -220,7 +220,7 @@ jobs:
 
     - name: BuildArguments
       value: >-
-        -subset host+packs -ci
+        -subset host+libs.pretest+packs -ci
```

Comment:
Hmm, this change surprises me. Why would we need libs.pretest in the installer partition?